src/App.js

```diff
@@ -3,14 +3,28 @@ import Homepage from "./components/Homepage"
 import Header from "./components/Header"
 import Jumbotron from './components/Jumbotron'
 import Footer from "./components/Footer"
+import Order_page from "./components/Order_page";
 
 const App = () => {
+  const [activeRestaurantName, setActiveRestaurantName] = useState("")
+  const [activeRestaurantId, setActiveRestaurantId] = useState(0)
+
+  function buttonSetter(name, id) {
```

Comment:
I don't love this function name, I think something like `setActiveRestaurant` would be a more descriptive name